internal/provider/resource_gitlab_group_label.go

```diff
@@ -72,7 +72,6 @@ func resourceGitlabGroupLabelCreate(ctx context.Context, d *schema.ResourceData,
 	}
 
 	d.SetId(label.Name)
```

Comment:
Not quite related to this PR, but I wonder if this could be problematic using the `Name` instead of the `ID` if someone were to rename the label.